.github/workflows/ci.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       contents: read
     strategy:
       matrix:
-        node-version: [18.17, 20, 22]
+        node-version: [18.19, 20, 22]
```

Comment:
isn't that technically a semver major? I'd rather use the opportunity to drop node 18 from the list altogether